services/ui-src/src/forms/mcpar/dpc/dpc.json

```diff
@@ -32,6 +26,26 @@
           "hint": "What is the total number of individuals enrolled in each plan as of the first day of the last month of the reporting year?",
           "mask": "comma-separated"
         }
+      },
+      {
+        "id": "dpc-2",
+        "type": "number",
+        "props": {
+          "label": "D1.I.2 Plan share of Medicaid",
+          "hint": "Automatically calculated. Sum of plan enrollment (within the specific program) as a percentage of the state's total Medicaid enrollment.",
+          "mask": "comma-separated",
```

Comment:
Remove mask for now. Will make percentage mask later. Applies to both `dpc-2` and `dpc-3`.
```suggestion
```